### ISSUE TITLE
[0.8.3.3] Compostable items turn into trash

### ISSUE THREAD

**Issue by GauHelldragon** [StrangeLoopGames/EcoIssues#16108]
Version 0.8.3.3 Beta
When creating trash piles and filling more than one item slot, the item will be turned into garbage even if it compostable ( a green background ) 

Testing: Created a Trash pile with 1 slot of 1 rice, one trash pile with 2 slots of 1 rice, one trash pile with 3 slots of 1 rice and one trash pile with 4 slots of rice

Start:
https://steamuserimages-a.akamaihd.net/ugc/1012691401816699006/905EAF97877CF2A649366DE282847CA379A0C93E/

Composting:
https://steamuserimages-a.akamaihd.net/ugc/1012691401817639519/478D7E821099047C4DDC79863F088772055F17D6/

Finish: 
https://steamuserimages-a.akamaihd.net/ugc/1012691509328150287/6CD4E61562EB5E9DD47E536B5E78D2C19A17F831/

The end result from 4 blocks of trash is 6 blocks of garbage and 3 blocks of compost. The only item used was rice.

**Comment by SlayksWood** [comment]
Thank you for the report. 
Can't reproduce in 0.9.0 staging-1536

**Comment by SlayksWood** [comment]
So can't reprocude in 0.9.0 stagings. Closed for now.
![image](https://user-images.githubusercontent.com/45708377/83601640-a2d95780-a579-11ea-8562-e2e85df5c6d6.png)

**Comment by Selenice** [comment]
It happenned to me again in 0.9.0.0 beta staging stable 8.
Created a trash pile with 4 full slots and 3 of the blocks turned into garbage, only one turned into compost. 
I used seeds. I will try to do more tests with 1, 2, and 3 full slots.
![trash](https://user-images.githubusercontent.com/68972491/88854702-f4a50380-d1f1-11ea-8dcb-7031e9a2acdd.png)
Here is the picture with the 3 trash blocks, the compost block was removed before the screen.
